app/controllers/mock-login.js

```diff
@@ -0,0 +1,32 @@
+import Controller from '@ember/controller';
+import { task, timeout } from 'ember-concurrency';
+import { set } from '@ember/object';
+
+export default class MockLoginController extends Controller {
+  page = 0;
+  size = 10;
+
+  @task
```

Comment:
I think the tasks can be removed. They seem to be related to a search input, which isn't needed in this app.